<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Background Tasks</string>
      </attribute>
      <widget class="QGroupBox" name="Autosave_groupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>331</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Auto-save</string>
       </property>
       <widget class="QLineEdit" name="auto_save_line">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>50</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QToolButton" name="auto_save_browse_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>50</y>
          <width>25</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="auto_save_checkbox">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>14</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Display</string>
      </attribute>
      <widget class="QGroupBox" name="results_groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>331</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Results view</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="plot_groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>331</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Plot view</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>API</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>auto_save_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>auto_save_line</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>auto_save_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>auto_save_browse_button</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
